src/data/customer/__test__/driver.test.js

```diff
@@ -15,9 +15,11 @@ import {
   getDriverRoutesAsync,
```

Comment:
@vdanylov please rebase with develop and rerun + commit the snapshots so the Travis CI can pass